From: Michael Davidsaver <mdavidsaver@gmail.com>
Date: Wed, 9 Mar 2016 16:10:35 -0500
Subject: Don't build test*

These aren't automatic tests we can run,
and won't install them, so don't bother
to build.
---
 Makefile | 48 ------------------------------------------------
 1 file changed, 48 deletions(-)

diff --git a/Makefile b/Makefile
index f2bbc8c..f2dff97 100644
--- a/Makefile
+++ b/Makefile
@@ -9,57 +9,9 @@ asyn_DEPEND_DIRS = configure
 DIRS += asyn/asynPortDriver/unittest
 asyn/asynPortDriver/unittest_DEPEND_DIRS = asyn
 
-ifneq ($(EPICS_LIBCOM_ONLY),YES)
-  DIRS += testApp
-  testApp_DEPEND_DIRS = asyn
-  iocBoot_DEPEND_DIRS += testApp
-  DIRS += testArrayRingBufferApp
-  testArrayRingBufferApp_DEPEND_DIRS = asyn
-  iocBoot_DEPEND_DIRS += testArrayRingBufferApp
-  DIRS += testAsynPortDriverApp
-  testAsynPortDriverApp_DEPEND_DIRS = asyn
-  iocBoot_DEPEND_DIRS += testAsynPortDriverApp
-  DIRS += testBroadcastApp
-  testBroadcastApp_DEPEND_DIRS = asyn
-  iocBoot_DEPEND_DIRS += testBroadcastApp
-  DIRS += testConnectApp
-  testConnectApp_DEPEND_DIRS = asyn
-  iocBoot_DEPEND_DIRS += testConnectApp
-  DIRS += testEpicsApp
-  testEpicsApp_DEPEND_DIRS = testApp asyn
-  iocBoot_DEPEND_DIRS += testEpicsApp
-  DIRS += testErrorsApp
-  testErrorsApp_DEPEND_DIRS = asyn
-  iocBoot_DEPEND_DIRS += testErrorsApp
-  DIRS += testGpibApp
-  testGpibApp_DEPEND_DIRS = testApp asyn
-  iocBoot_DEPEND_DIRS += testGpibApp
-  DIRS += testGpibSerialApp
-  testGpibSerialApp_DEPEND_DIRS = testApp testGpibApp
-  iocBoot_DEPEND_DIRS += testGpibSerialApp
-  ifdef SNCSEQ
-    DIRS += testIPServerApp
-    testIPServerApp_DEPEND_DIRS = asyn
-    iocBoot_DEPEND_DIRS += testIPServerApp
-  endif
-  DIRS += testManagerApp
-  testManagerApp_DEPEND_DIRS = asyn
-  iocBoot_DEPEND_DIRS += testManagerApp
-  DIRS += testOutputReadbackApp
-  testOutputReadbackApp_DEPEND_DIRS = asyn
-  iocBoot_DEPEND_DIRS += testOutputReadbackApp
   DIRS += testOutputCallbackApp
   testOutputCallbackApp_DEPEND_DIRS = asyn
   iocBoot_DEPEND_DIRS += testOutputCallbackApp
-  DIRS += testUsbtmcApp
-  testUsbtmcApp_DEPEND_DIRS = asyn
-  iocBoot_DEPEND_DIRS += testUsbtmcApp
-  DIRS += iocBoot
-endif
-
-DIRS += testAsynPortClientApp
-testAsynPortClientApp_DEPEND_DIRS = asyn
-
 include $(TOP)/configure/RULES_TOP
 
 UNINSTALL_DIRS := $(filter-out $(INSTALL_HTML),$(UNINSTALL_DIRS))
